Networking/NetworkingTests/ApplicationPassword/ApplicationPasswordEncoderTests.swift

```diff
@@ -1,5 +1,6 @@
 import XCTest
 @testable import Networking
+@testable import NetworkingCore
```

Comment:
It was simpler to add `@testable import Networking` to all the tests than to figure out failed build after failed build which tests needed it and which didn't.